target/linux/realtek/base-files/etc/board.d/02_network

```diff
@@ -69,6 +69,18 @@ done
 [ -n "$label_mac" ] && ucidef_set_label_macaddr $label_mac
 
 case $board in
+d-link_dgs-1210-10mp-f)
+	ucidef_set_poe 130 "$lan_list" "lan9 lan10"
+	;;
+d-link_dgs-1210-10p)
+	ucidef_set_poe 65 "$lan_list" "lan9 lan10"
+	;;
+d-link_dgs-1210-28mp-f)
+	ucidef_set_poe 370 "$lan_list" "lan25 lan26 lan27 lan28"
```

Comment:
I'm pretty sure that it's not a 1:1 mapping on this switch. I don't remember the exact sequence, but judging from my commit message https://github.com/openwrt/openwrt/commit/3e7e4d0b97595fc0c24f1d702de3daac2de3c2e7, every 8 ports need to be reversed.

I will check on my switch, but that's going to take a while until I have access to the switch.

Reply:
Playing around a bit, I think we might be able to generate this from $lan_list_rev:
```
$ for i in {15..0}; do echo lan$i; done | sort -V -r | xargs -n4 | sort -V | xargs
lan3 lan2 lan1 lan0 lan7 lan6 lan5 lan4 lan11 lan10 lan9 lan8 lan15 lan14 lan13 lan12
```

Reply:
I'm pretty certain that $lan_list_rev also contains lan25, lan26, lan27 and lan28. This will break the algorithm as every 8 ports need to be grouped, not just 4. 

I cannot flash/ramboot my switch right now, but I had a look at a working PoE config:

```
config port                                                                                                                                                          
        option enable   '1'                                                                                                                                          
        option id       '1'                                                                                                                                          
        option name     'lan8'                                                                                                                                       
        option poe_plus '1'                                                                                                                                          
        option priority '2'                                                                                                                                          
                                                                                                                                                                     
config port                                                                                                                                                          
        option enable   '1'                                                                                                                                          
        option id       '2'                                                                                                                                          
        option name     'lan7'                                                                                                                                       
        option poe_plus '1'                                                                                                                                          
        option priority '2'                                                                                                                                          
                                                                                                                                                                     

...

config port                                                                                                                                                          
        option enable   '1'                                                                                                                                          
        option id       '9'                                                                                                                                          
        option name     'lan16'                                                                                                                                      
        option poe_plus '1'                                                                                                                                          
        option priority '2'                                                                                                                                          
                                                                                                                                                                     
config port                                                                                                                                                          
        option enable   '1'                                                                                                                                          
        option id       '10'                                                                                                                                         
        option name     'lan15'                                                                                                                                      
        option poe_plus '1'                                                                                                                                          
        option priority '2'       

...

config port                                                                                                                                                          
        option enable   '1'                                                                                                                                          
        option id       '17'                                                                                                                                         
        option name     'lan24'                                                                                                                                      
        option poe_plus '1'                                                                                                                                          
        option priority '2'                                                                                                                                          
                                                                                                                                                                     
config port                                                                                                                                                          
        option enable   '1'                                                                                                                                          
        option id       '18'                                                                                                                                         
        option name     'lan23'                                                                                                                                      
        option poe_plus '1'                                                                                                                                          
        option priority '2'                                  
```

Furthermore, on a 23.05.2 build, the board should read `d-link,dgs-1210-28mp-f`. Is this a mistake or was it renamed in current snapshot?

Reply:
Thanks @andyboeh  that was a copy/paste error.  Name comes from the dts file not the image name in make file!  I squashed a fix along with an attempt to properly define the DGS-1210-28MP.